cargo-screeps/README.md

```diff
@@ -46,7 +46,61 @@ This is not fully tested, but feel free to use! Issues are welcome.
   - runs `cargo web --check --target=wasm32-unknown-unknown` which is fairly similar to
     `cargo check`.
 
+# Configuration Options
```

Comment:
Might be worthwhile to start the document with this section as it is required to do deploy and check from the `Build Options` section.

Reply:
Still, `build` is usable without it and I'd like to start upfront with what `cargo screeps` can do.

What would you think of my adding a note at the start of each command mentioning what configuration sections it requires?

Reply:
I've mentioned what each command required here: https://github.com/daboross/screeps-in-rust-via-wasm/pull/85/commits/c532ece6fd294c4b3aa984f7fdbc18ddb4cf5a79.

Both `build` and `check` commands should be fully usable with a blank `screeps.toml` though.

I mean if you think it would be best, I can rearrange the sections.